frontend/src/pages/secret-manager/SecretDashboardPage/SecretDashboardPage.tsx

```diff
@@ -137,6 +141,11 @@ const Page = () => {
     subject(ProjectPermissionSub.DynamicSecrets, { environment, secretPath })
   );
 
+  const canReadSecretRotations = permission.can(
```

Comment:
was also able to rollback mapped secrets via PIT recovery

<img width="1512" alt="image" src="https://github.com/user-attachments/assets/cf166a2c-9e9e-46de-8e87-339d29aff9f1" />